src/configs/settings.ts

```diff
@@ -882,21 +881,21 @@ export const indicatorThresholdsIRSNamibia: IndicatorThresholds = {
     name: IRS_GREEN_THRESHOLD,
     value: 1,
   },
-  LIGHT_GREEN_THRESHOLD: {
-    color: '#76BA1B',
-    name: IRS_LIGHT_GREEN_THRESHOLD,
-    value: 0.85,
+  GREY_THRESHOLD: {
+    color: '#dddddd',
+    name: IRS_GREY_THRESHOLD,
+    value: 0.2,
   },
   RED_THRESHOLD: {
     color: '#FF4136',
     name: IRS_RED_THRESHOLD,
     orEquals: true,
```

Comment:
Should this still have orEquals as true?

Reply:
@moshthepitt Not sure but it is still the same as the default thresholds above it. Also, doesn't affect the component `src/components/formatting/IRSIndicatorLegend/index.tsx`. The `orEquals` for the default threshold affects component `src/helpers/ProgressBar.tsx`